### ISSUE TITLE
Crash report from Dynamo 2.5.0.7460

### ISSUE THREAD

**Issue by farshidkaviani** [DynamoDS/Dynamo#10895]
# Issue Description
Please fill in the following information to help us reproduce the issue:

### What did you do?
(Fill in here)

### What did you expect to see?
(Fill in here)

### What did you see instead?
(Fill in here)

### What packages or external references (if any) were used?
(Fill in here)

### Stack Trace
```
(From the Dynamo crash window select 'Details' -> 'Copy' and paste here)
```

---
OS: `Microsoft Windows NT 10.0.18363.0`
CLR: `4.0.30319.42000`
Dynamo: `2.5.0.7460`

**Comment by Amoursol** [comment]
Hello @farshidkaviani - Are you able to please edit your post and fill in the template? We need more information on what happened, what you were trying to do and if possible an uploaded .DYN file for this so we can look to fix it!

**Comment by hikozan** [comment]
I am sorry. I did not know how I reported the thing by this dialog.
The crash happed when I connected the node "Material.StructuralParameters" to the node "String from Object".
This operation was O.K. in the Dynamo 2.0.3 but not in the Revit2019 and Revit2020.
Maybe something of Material.StructuralParameters has been changed.

**Comment by Amoursol** [comment]
Hello @farshidkaviani - - As there has been no additional information provided we are closing this for now. Please re-open and fill in the questions asked in the template if this issue persists.

**Comment by hikozan** [comment]
Dear Sol Amour,
How can I use the Dynamo 2.5.0.7460.
I asked a sales person of AUTODESK but he just said “ask forum”.

Hikozan
Tadahiko SHIOMI

**Comment by Amoursol** [comment]
Hello @hikozan - `Dynamo 2.5` is available in `Revit 2021` (`Revit 2021.1 `now has `Dynamo 2.6.1`). If you are working in Revit 2018, or Revit 2019 our final installer version is `Dynamo 2.0.4` which we released a month ago 😊 

If you want to use **Dynamo Sandbox**, you can download it from dynamobuilds.com
